apps/api/src/app/inbox/usecases/mark-many-notifications-as/mark-many-notifications-as.usecase.ts

```diff
@@ -178,5 +179,6 @@ function createTraceLog({
     status: 'success',
     entity_type: 'step_run',
     entity_id: message._jobId,
+    stp_step_type: message.channel as StepType,
```

Comment:
what does `stp` means? I find it a little confusing, maybe just step type?

Reply:
Let's rename it to step_run_type

Also we need workflow_run_identifier

Reply:
done